pkg/beyla/config.go

```diff
@@ -39,7 +39,7 @@ const (
 )
 
 const (
-	defaultMetricsTTL = 70 * time.Minute
+	defaultMetricsTTL = 5 * time.Minute
```

Comment:
Expiration works now correctly, we don't need the insanely high interval anymore